openpype/hosts/traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -0,0 +1,682 @@
+import os
+import re
+import csv
+import clique
+from copy import deepcopy, copy
+
+from openpype.client import get_asset_by_name
+from openpype.pipeline.create import get_subset_name
+from openpype.pipeline import CreatedInstance
+from openpype.lib import FileDef, BoolDef
+from openpype.lib.transcoding import (
+    VIDEO_EXTENSIONS, IMAGE_EXTENSIONS
+)
+
+from openpype.hosts.traypublisher.api.plugin import (
+    TrayPublishCreator
+)
+
+
+class IngestCSV(TrayPublishCreator):
+    """CSV ingest creator class"""
+
+    icon = "fa.file"
+
+    label = "CSV Ingest"
+    family = "csv_ingest_file"
+    identifier = "io.openpype.creators.traypublisher.csv_ingest"
+
+    default_variants = ["Main"]
+
+    description = "Ingest products' data from CSV file"
+    detailed_description = """
+Ingest products' data from CSV file following column and representation
+configuration in project settings.
+"""
+
+    # Position in the list of creators.
+    order = 10
+
+    # settings for this creator
+    column_config = {}
```

Comment:
The apply_settings method is not supported? How what is the proper way of reading the settings?

_WARNING: Source - Create plugin EditorialSimpleCreator. System settings argument will not be passed to '__init__' and 'apply_settings' methods in future versions of OpenPype._

```
    def apply_settings(self, project_settings):
        creator_settings = (
            project_settings["traypublisher"]["create"]["IngestCSV"]
        )
        self.column_config = creator_settings["columns_config"]
        self.representation_config = creator_settings["representations_config"]
```

Reply:
dev of this feature were moved into https://github.com/ynput/ayon-core/pull/100